Makefile

```diff
@@ -110,14 +110,14 @@ accept-e2e-regression-suite:
 ifndef SUITE
 	$(error SUITE is undefined)
 endif
-	[[ -d ./tests/e2e_regression/$(SUITE)/actual ]] || { echo "Note: No actual outputs found for suite $(SUITE). Nothing to accept."; exit 0; } \
-	# Delete all old expected files first, in case any test cases were renamed or removed. \
+	[[ -d ./tests/e2e_regression/$(SUITE)/actual ]] || { echo "Note: No actual outputs found for suite $(SUITE). Nothing to accept."; exit 0; }
+	# Delete all old expected files first, in case any test cases were renamed or removed.
 	rm -rf ./tests/e2e_regression/$(SUITE)/expected; \
-	# Copy the actual outputs to the expected outputs. \
+	# Copy the actual outputs to the expected outputs.
 	cp -r  ./tests/e2e_regression/$(SUITE)/{actual,expected}; \
-	# The result of the "spec" test is a special case. It should always match the \
-	# current openAPI spec file, so we symlink it to avoid having to update the expected \
-	# output every time the spec changes. \
+	# The result of the "spec" test is a special case. It should always match the
+	# current openAPI spec file, so we symlink it to avoid having to update the expected
+	# output every time the spec changes.
```

Comment:
I think the `\` at the end of commented lines caused subsequent non-comment lines to be treated as a comment and skipped

Reply:
what was the intention with these? run the whole thing in one shell instead of several?

Reply:
hm I'm not sure.. maybe @mitjat  knows? I'm also curious as to whether it works locally for y'all as-is

Reply:
just tried on linux with GNU bash, version 5.2.21(1)-release and GNU Make 4.3, the lower lines seem to run normally

Reply:
Interesting. I don't remember why the `\` are there, unfortunately. If this works locally and in CI, LGTM.

Although I'd say:
- If we're doing this, let's remove all `\` in this recipe so we're not using a weird mix.
- Prepend every `#` with `@` or else `make` will echo those comments to the console every time we run the recipe.

Reply:
this is moved to a new script e2e_regression/accept.sh, which makes it no longer necessary to use `\`s. it also should quiet down the logging of every line